cmd/skywire-cli/commands/config/gen.go

```diff
@@ -4,55 +4,134 @@ package cliconfig
 import (
 	"encoding/json"
 	"fmt"
+	"io"
+	"net"
+	"net/http"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 	"strings"
+	"time"
 
 	"github.com/bitfield/script"
-	"github.com/sirupsen/logrus"
+	"github.com/skycoin/dmsg/pkg/disc"
+	"github.com/skycoin/dmsg/pkg/dmsgpty"
+	coinCipher "github.com/skycoin/skycoin/src/cipher"
 	"github.com/spf13/cobra"
 
+	"github.com/skycoin/skywire-utilities/pkg/buildinfo"
 	"github.com/skycoin/skywire-utilities/pkg/cipher"
-	"github.com/skycoin/skywire-utilities/pkg/logging"
 	"github.com/skycoin/skywire-utilities/pkg/netutil"
+	utilenv "github.com/skycoin/skywire-utilities/pkg/skyenv"
 	"github.com/skycoin/skywire/pkg/app/appserver"
+	"github.com/skycoin/skywire/pkg/dmsgc"
+	"github.com/skycoin/skywire/pkg/restart"
 	"github.com/skycoin/skywire/pkg/routing"
 	"github.com/skycoin/skywire/pkg/skyenv"
+	"github.com/skycoin/skywire/pkg/transport/network"
 	"github.com/skycoin/skywire/pkg/visor/visorconfig"
 )
 
+// RootCmd contains commands that interact with the config of local skywire-visor
+var checkPKCmd = &cobra.Command{
+	Use:   "check-pk <public-key>",
+	Short: "check a skywire public key",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			return
+		}
+		var checkKey cipher.PubKey
+		err := checkKey.Set(args[0])
+		if err != nil {
+			logger.WithError(err).Fatal("invalid public key ") //nolint
+		}
+		logger.Info("Valid public key: ", checkKey.String())
+	},
+}
+
+// RootCmd contains commands that interact with the config of local skywire-visor
+var genKeysCmd = &cobra.Command{
+	Use:   "gen-keys",
+	Short: "generate public / secret keypair",
+	Run: func(cmd *cobra.Command, args []string) {
+		pk, sk := cipher.GenerateKeyPair()
+		fmt.Println(pk)
```

Comment:
Can we add labels here? So someone not familiar with Skycoin keys can see which one is private and which one is the public key? 

`Public key: $pk`
`Secret key: $sk`